lib/broccoli/ember-app.js

```diff
@@ -256,7 +256,7 @@ EmberApp.prototype._initVendorFiles = function() {
     handlebarsVendorFiles = null;
   }
 
-  this.vendorFiles = omitBy(merge({
+  this.vendorFiles = omit(merge({
```

Comment:
I do not believe that we have `omit` being required any more, I believe it should be `omitBy` and this sholdn't need to be changed...